tests/test_symbolic_coefficients.py

```diff
@@ -138,6 +242,7 @@ def test_staggered_array(self, order):
 
         eq_f = Eq(f, f.dx2, coefficients=Substitutions(coeffs_f))
         eq_g = Eq(g, g.dx2, coefficients=Substitutions(coeffs_g))
+        # Evaluate and check against one another?
```

Comment:
?

Reply:
Comments I left for myself when writing and forgot to clean up. Will remove